server/src/document/document.controller.ts

```diff
@@ -70,9 +70,10 @@ export class DocumentController {
     }
   }
 
+  
   @Get('/*')
-  async read(@Param() path, @Res() res) {
-    const stream = await this.documentService.getPublicPackageOrArtifactDocument(path);
+  async read(@Param() sharepointRelativePath, @Res() res) {
+    const stream = await this.documentService.getDocument(sharepointRelativePath[0]);
```

Comment:
Just curious about the sharepoint process since I'm a bit unfamiliar -- the `sharepointRelativePath` comes through as an array?

Reply:
Yeah, the `@Param` decorator returns an array. I thought it was as bit weird, too. I think it's an abstraction to handle some more complex URLs?